src/qt/test/wallettests.cpp

```diff
@@ -68,6 +68,18 @@ QModelIndex FindTx(const QAbstractItemModel& model, const uint256& txid)
 }
 
 //! Simple qt wallet tests.
+//
+// Test widgets can be debugged interactively calling show() on them and
+// manually running the event loop, e.g.:
```

Comment:
>Can this be done automatically

It would probably be possible to test code that would show all widgets. So far I've found it useful just to show selected widgets when debugging.